crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/config/MaatApiOAuth2Client.java

```diff
@@ -142,6 +143,11 @@ private ExchangeFilterFunction errorResponse() {
                                 errorMessage
                         );
                     }
+                    if (r.statusCode().equals(HttpStatus.NOT_FOUND)) {
+                        return WebClientResponseException.create(
+                                r.rawStatusCode(), r.statusCode().getReasonPhrase(), null, null, null
+                        );
```

Comment:
I would not have these line breaks for closing brackets. Same comment goes for other classes as well